README.md

```diff
@@ -126,7 +129,10 @@ var ast = espree.parse(code, {
         jsx: true,
 
         // enable return in global scope
-        globalReturn: true
+        globalReturn: true,
+
+        // allow experimental object rest/spread
+        experimentalObjectRestSpread: true
```

Comment:
Would this be better as an object like this:

```js
experimental: {
       ObjectRestSpread: true
}
```

So that its easy to add more and they are all under the same blovk